stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/viewmodel/messages/MessageListViewModel.kt

```diff
@@ -170,7 +178,6 @@ public class MessageListViewModel(
      * Holds the current [MessageMode] that's used for the messages list. [MessageMode.Normal] by default.
      */
     public var messageMode: MessageMode by mutableStateOf(MessageMode.Normal)
-        private set
```

Comment:
why did we remove this?

Reply:
forgot to add it back, was testing how it would work with a controller which is to be implemented later.